subsys/bluetooth/host/hci_core.c

```diff
@@ -194,6 +194,91 @@ static void handle_vs_event(uint8_t event, struct net_buf *buf,
 	/* Other possible errors are handled by handle_event_common function */
 }
 
+const char *bt_hci_err_to_str(uint8_t hci_err)
+{
+	const char * const mapping_table[] = {
+		[BT_HCI_ERR_SUCCESS] = "BT_HCI_ERR_SUCCESS",
```

Comment:
If the string is just going to be the preprocessor define as a string, I guess you could wrap this into a macro to avoid duplication, something like:
```c
#define HCI_ERR(err) [err] = STRINGIFY(err)
```

And then the table becomes:
```c
	const char * const mapping_table[] = {
		HCI_ERR(BT_HCI_ERR_SUCCESS),
		HCI_ERR(BT_HCI_ERR_UNKNOWN_CMD),
		...
```

Reply:
Seems like checkpatch was not too happy about this change. Is there something that can be done to mark this as accepted?